src/components/Output.js

```diff
@@ -11,14 +11,16 @@ export function Output({
 }) {
   return babelConfig.map((config, index) => {
     return (
-      <CompiledOutput
-        source={debouncedSource}
-        customPlugin={enableCustomPlugin ? customPlugin : undefined}
-        config={config}
-        key={index}
-        onConfigChange={config => updateBabelConfig(config, index)}
-        removeConfig={() => removeBabelConfig(index)}
-      />
+      <Fragment>
+        <CompiledOutput
+          source={debouncedSource}
+          customPlugin={enableCustomPlugin ? customPlugin : undefined}
+          config={config}
+          key={index}
+          onConfigChange={config => updateBabelConfig(config, index)}
+          removeConfig={() => removeBabelConfig(index)}
+        />
+      </Fragment>
```

Comment:
Why does this need to be wrapped in a `Fragment`?

Reply:
Ah thanks for reminding! I had previously moved TimeTravel out to be siblings with CompiledOutput, so Fragment would just couple them without producing a `<div>`-like tag, I'll remove it